controlpanel/frontend/views/dashboard.py

```diff
@@ -108,12 +108,19 @@ def get_context_data(self, **kwargs):
         dashboard = self.get_object()
         dashboard_admins = dashboard.admins.all()
 
-        context["admin_options"] = User.objects.exclude(
+        potential_admins = User.objects.exclude(
             auth0_id="",
         ).exclude(
             auth0_id__in=[user.auth0_id for user in dashboard_admins],
         )
 
+        context["admin_options"] = [
+            user
+            for user in potential_admins
+            if user.has_perm("api.quicksight_embed_author_access")
```

Comment:
We should probably check for these permissions here also https://github.com/ministryofjustice/analytics-platform-control-panel/blob/main/controlpanel/frontend/views/dashboard.py#L149

FYI I think that change will break [this test](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/main/tests/frontend/views/test_dashboard.py#L504) so the fixtures may need to be updated, to add a QS user